pavlo976/BuildMinder-App
# Feature/basic page routes

### What changed?
React router basic browser routes was created 
### How to test it?
Open application and try to write pathes in url bar, i.e: 'yourdomain/path and check that the page opens properly
### What's missing?
Everything is up to date
